zeppelin-server/src/main/java/org/apache/zeppelin/rest/LoginRestApi.java

```diff
@@ -126,8 +123,9 @@ public Response logout() {
     data.put("principal", "anonymous");
     data.put("roles", "");
     data.put("ticket", "anonymous");
+    data.put("WWW-Authenticate", "Basic realm=\"Login required\"");
```

Comment:
Yes, you are right, let me put `WWW-Authenticate` in response header.

I was already doing this, but it was not solving the case for basic-authentication. Let me try your recommendation, I'll update in a bit.
```
Subject currentUser = org.apache.shiro.SecurityUtils.getSubject();
currentUser.logout();
```